webapp/src/app/components/dialogs/settings/settings.component.html

```diff
@@ -13,12 +13,22 @@
 				</mat-form-field>
 				<div class="actions-container" fxLayout="row">
 					<button mat-stroked-button (click)="select()">Select</button>
-					<button mat-stroked-button (click)="check()">
+					<button mat-stroked-button (click)="check() && initMods()">
 						Check
 						<mat-icon [ngClass]="iconCss" class="icon-base">{{icon}}</mat-icon>
 					</button>
 				</div>
 			</form>
+			<form fxLayout="row" class="option-row">
+				<mat-form-field fxFlex="auto">
+					<mat-label>Map Context</mat-label>
```

Comment:
`Map Context` can be a bit confusing because it isn't directly related to modding. Maybe use `Mod`, `Mod map filter` or similar